csg.js

```diff
@@ -6129,14 +6129,44 @@ for solid CAD anyway.
                 for (var ii = i + 1; ii < numsides; ii++) {
                     var side1 = this.sides[ii];
                     if (CAG.linesIntersect(side0.vertex0.pos, side0.vertex1.pos, side1.vertex0.pos, side1.vertex1.pos)) {
-                        if (debug) { OpenJsCad.log(side0); OpenJsCad.log(side1);}
+                        if (debug) { console.log("side "+i+": "+side0); console.log("side "+ii+": "+side1);}
                         return true;
                     }
                 }
             }
             return false;
         },
 
+        fixClosure: function() {
```

Comment:
I can take this out of this PR.

This function fixes an issue that I found with the union() function. See issue #15 
Basically, CSG.union() is broken, leaving a gap in the polygons. When converting back to CAG, that leaves a gap in the sides. This routine detects and fixes a gap.

I don't like it but...